docs/resources/oracle_dsource.md

```diff
@@ -0,0 +1,197 @@
+# Resource: <resource name> delphix_oracle_dsource
+
+In Delphix terminology, a dSource is a database that the Delphix Continuous Data Engine uses to create and update virtual copies of your database. 
+A dSource is created and managed by the Delphix Continuous Data Engine.
+
+The dSource resource allows Terraform to create and delete Delphix dSources. This specifically enables the apply and destroy Terraform commands. Modification of existing dSource resources via the apply command is not supported. All supported parameters are listed below.
+
+## System Requirements
+
+* Data Control Tower v10.0.1+ is required for dSource management. Lower versions are not supported.
+* The dSource Resource does not support SQL Server, or SAP ASE. The below examples are shown from the Oracle context.
+
+## Example Usage
+
+The linking of a dSource can be configured through various ingestion approaches. Each configuration is customized to the connector and its supported options. The three PostgreSQL parameter sets below show working examples.
```

Comment:
"The three PostgreSQL parameter sets below show working examples."

I only see one? And its for Oracle

Reply:
>The linking of a dSource can be configured through various ingestion approaches.

I think we are demonstrating the "Direct Ingestion" approach in our below example? Let's call that out.